modules/back-end/src/Application/EndUsers/GetFeatureFlagEndUserList.cs

```diff
@@ -0,0 +1,75 @@
+using Application.Bases;
+using Application.Bases.Models;
+using Domain.FeatureFlags;
+
+namespace Application.EndUsers;
+
+public class GetFeatureFlagEndUserList : IRequest<PagedResult<FeatureFlagEndUserStatsVm>>
+{
+    public Guid EnvId { get; set; }
+
+    public FeatureFlagEndUserFilter Filter { get; set; }
+}
+
+public class GetFeatureFlagEndUserListValidator : AbstractValidator<GetFeatureFlagEndUserList>
+{
+    public GetFeatureFlagEndUserListValidator()
+    {
+        RuleFor(x => x.Filter.FeatureFlagKey)
+            .NotEmpty().WithErrorCode(ErrorCodes.FeatureFlagKeyIsRequired);
+
+        RuleFor(x => x.Filter.From)
+            .GreaterThan(0).WithErrorCode(ErrorCodes.StatsFromIsRequired);
+
+        RuleFor(x => x.Filter.To)
+            .GreaterThan(0).WithErrorCode(ErrorCodes.StatsToIsRequired);
```

Comment:
not a big problem, but i think the error code should be `ErrorCodes.InvalidStatsFrom` and `ErrorCodes.InvalidStatsTo`.

Reply:
same for `GetStatsByVariationValidator`